### PR TITLE
Add axis+ticks+grid

### PR DESCRIPTION
**Pull request by almarklein**
Starting to look like a proper plot. When the ticks were shown, I easily spotted a bug that it also drew a line `t -> t`. Tools not yet working.

### CODE REVIEW

File: data/bokeh/bokehhelper.js [hunk 1]
```diff
@@ -1,76 +1,53 @@
...
+        return renderer;
+    }
+    function add_tools(plot, tools) {
+        for (var i=0; i<tools.length; i++) {
+            var tool = tools[i];
+            tool.set('plot', plot);            
         }
-        return renderers;
+        plot.set("tools", plot.get("tools").concat(tools));
+        // TODO: tools not shown: need more?
```
Comment (almarklein) [inline data/bokeh/bokehhelper.js:30]: @bryevdv ideas?

Comment (almarklein) [inline data/bokeh/bokehhelper.js:30]: (maybe best to view this file as a whole)

Comment (bryevdv) [inline data/bokeh/bokehhelper.js:30]: maybe need to set `toolbar_location`? not sure what its default is

Comment (bryevdv) [inline data/bokeh/bokehhelper.js:30]: that would be the case if e.g. tools work, there is just no toolbar

**Comment by dwillmer** [comment]
:+1:

**Pull request merged**